### PR TITLE
93 start prototyping ml model for rq4

### LINKED ISSUE #93: Start prototyping ML model for RQ4
#### Given a target final cumulative reward and specific hardware, can a model infer the necessary hyperparameters (including training steps) to achieve that performance within a specified confidence interval?

### PR DESCRIPTION
**Pull request by PedroPianna**
### Proposed change(s)

Describe the changes made in this PR.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the changelog (if applicable)
- [ ] Updated the documentation (if applicable)
- [ ] Updated the migration guide (if applicable)

### CODE REVIEW

File: .env
Comment (PedroPianna) [inline .env:1]: Private keys or credentials should never be committed
